src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -56,10 +57,10 @@ public static int Run(ParseResult parseResult)
                 return ForwardToVSTestConsole(parseResult, args, settings, testSessionCorrelationId);
             }
 
-            return ForwardToMsbuild(parseResult, settings, testSessionCorrelationId);
+            return ForwardToMsbuild(parseResult, args, settings, testSessionCorrelationId);
         }
 
-        private static int ForwardToMsbuild(ParseResult parseResult, string[] settings, string testSessionCorrelationId)
+        private static int ForwardToMsbuild(ParseResult parseResult, string[] args, string[] settings, string testSessionCorrelationId)
```

Comment:
Forgot to cleanup this part:

```suggestion
        private static int ForwardToMsbuild(ParseResult parseResult, string[] settings, string testSessionCorrelationId)
```